pkg/asset/installconfig/ovirt/config.go

```diff
@@ -85,3 +87,33 @@ func (c *Config) Save() error {
 	}
 	return ioutil.WriteFile(path, out, 0600)
 }
+
+// Validate will validate if the provided certificate and credentials are valid. It will also issue log messages if the
+// credentials are not valid or the connection could otherwise not be established.
+func (c *Config) Validate() bool {
```

Comment:
Rather than logging in the `Validate` function, return an error and let the caller decide what to do with the error.